itests/utilities/reward/reward.go

```diff
@@ -83,3 +83,43 @@ func GetBlockRewardDistribution(suite *f.BaseSuite, addresses testdata.Addresses
 		diffDaoBalance.BalanceInWavesGo, diffDaoBalance.BalanceInWavesScala, diffXtnBalance.BalanceInWavesGo,
 		diffXtnBalance.BalanceInWavesScala), term
 }
+
+type TestData func(suite *f.BaseSuite, addresses testdata.AddressesForDistribution,
```

Comment:
Strange naming for the closure or function type. Is it testdata generator function type?

Reply:
This function type is used for common representation of function for getting test data. Renamed to GetTestData.